docroot/themes/humsci/humsci_basic/src/scss/components/_node--news-item.scss

```diff
@@ -65,7 +65,6 @@
   @include hb-colorful {
     $hb-color--primary: hb-colorful-variation(primary);
     position: relative;
-    margin-right: hb-calculate-rems(24px);
```

Comment:
Here is the original, the margin is being applied to the <a> tag but I believe the icon is ignoring that margin. It looks pretty broken to me:
<img width="396" alt="Screen Shot 2020-04-01 at 11 35 27 AM" src="https://user-images.githubusercontent.com/19614647/78156871-5a80aa80-740d-11ea-871c-adf42dfe5d55.png">

Here is the update. I removed the 24px margin right because it wasn't affecting the position of the icon with the link and looked less broken 🤷‍♀ . Maybe I'm missing something else here though. Let me know your thoughts.
<img width="392" alt="Screen Shot 2020-04-01 at 11 35 39 AM" src="https://user-images.githubusercontent.com/19614647/78157030-80a64a80-740d-11ea-8b2d-ec2514c9fb96.png">

Reply:
FYI Those are both in the same pixel screen width. I just removed the margin-right in the inspector.

Reply:
Yeah, I see that it would be nicer to have that space to reduce the possibility of wrapping, but the alternative is the possibility of icon overlapping. Here's an example of what I mean:

![image](https://user-images.githubusercontent.com/1256329/78160146-89991b00-7411-11ea-8197-16266dcdea8b.png)

It might be uncommon, but if we don't provide space for the icon then it could happen anywhere a news item link is displayed. The early wrapping is unfortunate, but I feel like your other change mitigates most wrapping. To me the wrapping looks a bit awkward, but it doesn't look broken, like overlapping icons do (see this example from the footer):

![image](https://user-images.githubusercontent.com/1256329/78161273-33c57280-7413-11ea-86b3-10f62da218d9.png)

Reply:
Okay makes sense, I can add back that margin. More than the wrapping of text what looks awkward is our hover state when it wraps:
<img width="362" alt="Screen Shot 2020-04-01 at 12 25 25 PM" src="https://user-images.githubusercontent.com/19614647/78161701-ded62c00-7413-11ea-8f1a-cda204c0912b.png">

Reply:
Ah I see. Thanks. I know it's not ideal, but feels like we're stuck between a rock and a hard place. This wouldn't be a problem if we used traditional underlines for links. Or if our external link icon situation wasn't so wonky, due to platform restrictions. But all these issues together kind of forces our hand.

Reply:
hm makes sense. maybe for inline and external links I can change the style to be underlined rather than a border bottom. I can add that fix to my next card.

Reply:
I think it would make sense to change inline and external links to be underlined rather than border bottom. I'm going to add that to my next polish card. For now I will keep the margin as it was.

Reply:
Oh! That could be nice. That sounds like a good topic for the design meeting too. 👍